### ISSUE TITLE
Use HTTPS instead of HTTP in urlset

### ISSUE THREAD

**Issue by TimoRuetten** [ekalinin/sitemap.js#164]
I have display issues in my Browsers when I want to test and see the generated XML file. It seems that you are using HTTP protocol instead of the HTTPS . When I changed it to HTTPS its working fine.

```xml
<urlset xmlns="https://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="https://www.w3.org/1999/xhtml" xmlns:mobile="https://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="https://www.google.com/schemas/sitemap-image/1.1" xmlns:video="https://www.google.com/schemas/sitemap-video/1.1">
```

**Comment by LukeOak** [comment]
Same problem here. When I try to use `links` my XML completely fall apart. Temporal work around
```
sm.createSitemap({
    xmlNs: 'xmlns="https://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="https://www.w3.org/1999/xhtml..."'
})
```

**Comment by ekalinin** [comment]
Will be fixed in #165
